js/stdlib/launchToken.mjs

```diff
@@ -4,15 +4,18 @@ import ETHstdlib from './stdlib_sol';
 import algosdk from 'algosdk';
 import real_ethers from 'ethers';
 import * as cfxers from './cfxers.mjs';
-// import type { EthersLike } from './ETH_like_interfaces';
 
-export default async function (name, sym) {
+export default async function (name, sym, stdlib_, accCreator_) {
```

Comment:
You should just change all the use cases to use this better way of doing things, since this is not documented and can be changed whenever we want